src/promptflow/promptflow/executor/flow_executor.py

```diff
@@ -1062,3 +1066,63 @@ def wrapper(*args, **kwargs):
         return result
 
     return wrapper
+
+
+def execute_flow(
+    flow_file: Path,
+    working_dir: Path,
+    output_dir: Path,
+    connections: dict,
+    inputs: Mapping[str, Any],
+    *,
+    func: Optional[str] = None,
```

Comment:
Maybe use **kwargs and bypassed to flow.create?

Reply:
Updated.